pkg/proto/types.go

```diff
@@ -4033,6 +4033,37 @@ type fieldsHashesJS struct {
 	LeaseBalanceHash  DigestWrapped `json:"leaseBalanceHash"`
 }
 
+func (s FieldsHashes) EqualWith(other FieldsHashes) bool {
```

Comment:
It can be even simpler, like:
```go
return s.DateEntryHash == other.DataEntryHash && s.AccountScriptHash == other.AccountScriptHash && ...
```
And name of the function better change to `Equal`.

Reply:
Usually argument `other` named as `o`.

Reply:
I would leave `other` as it is if that's okay. I don't like one-letter variables, they always make code hard to read.

Reply:
@esuwu First issue is still relevant.